File: Examples/CSharp/Articles/FitAllWorksheetColumns.cs
using System.IO;
using System;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Articles
{
    public class FitAllWorksheetColumns
    {
        public static void Run()
        {
            //Source directory
            string sourceDir = RunExamples.Get_SourceDirectory();

            //Output directory
            string outputDir = RunExamples.Get_OutputDirectory();

            // Create and initialize an instance of Workbook
            Workbook book = new Workbook(sourceDir + "sampleFitAllWorksheetColumns.xlsx");
            
            // Create and initialize an instance of PdfSaveOptions
            PdfSaveOptions saveOptions = new PdfSaveOptions(SaveFormat.Pdf);
            
            // Set AllColumnsInOnePagePerSheet to true
            saveOptions.AllColumnsInOnePagePerSheet = true;

            // Save Workbook to PDF fromart by passing the object of PdfSaveOptions
            book.Save(outputDir + "outputFitAllWorksheetColumns.pdf", saveOptions);            

            Console.WriteLine("FitAllWorksheetColumns executed successfully.\r\n");
        }
    }
}

File: Examples/VisualBasic/Articles/FitAllWorksheetColumns.vb
Imports Microsoft.VisualBasic
Imports System.IO

Imports Aspose.Cells

Namespace Articles
    Public Class FitAllWorksheetColumns
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            ' Create and initialize an instance of Workbook
            Dim book As New Workbook(dataDir & "TestBook.xlsx")
            ' Create and initialize an instance of PdfSaveOptions
            Dim saveOptions As New PdfSaveOptions(SaveFormat.Pdf)
            ' Set AllColumnsInOnePagePerSheet to true
            saveOptions.AllColumnsInOnePagePerSheet = True
            ' Save Workbook to PDF fromart by passing the object of PdfSaveOptions
            book.Save(dataDir & "output.pdf", saveOptions)
            ' ExEnd:1
        End Sub
    End Class
End Namespace